.vuepress/config.js

```diff
@@ -105,6 +105,23 @@ module.exports = {
             '/learn/core_concepts/relevancy',
           ],
         },
+        {
+          title: '⚙️ Configuration',
```

Comment:
The headings "Configuration" and "Index settings" aren't clickable. Adding the path should fix that
path: '/learn/advanced/instance_options', after `title: '⚙️ Configuration'`,
 path: '/learn/advanced/distinct', after ` title: 'Index settings'`

Reply:
Is it weird that I like the fact that it's not clickable? 🤔

Reply:
I think what we have here is a "worst of all possible worlds" scenario: these menu items have :hover effects, but are not actually clickable. So we're visually saying "hey, you can interact with this thing!" but not following up on our promise, as it were.

Though I align a bit more with @dichotommy on this one and don't think these items need to be clickable, I think @maryamsulemani97 is right. Clicking on every other main menu heading ("What is Meilisearch", "Core concepts", etc) leads to the first element of that menu section (e.g. "Core concepts" leads to "Documents"), so changing behaviour only for "Configuration" would make the interface less consistent and more confusing because the user.

Reply:
> Clicking on every other main menu heading ("What is Meilisearch", "Core concepts", etc) leads to the first element of that menu section (e.g. "Core concepts" leads to "Documents")...

This isn't really the case currently though, is it? Clicking on those navbar links you mentioned takes you to the `readme` for that directory (which I strongly dislike).

But yes, I agree with what you wrote: presenting unclickable menu items w/ a hover highlight is pretty unconscionable. I'm currently looking at the `config.js` file for VuePress's own documentation site to get a better sense of the "right way" to make section titles unclickable and get rid of some of our readmes.